terraform/aws/environments/dev/cluster/variables.tf

```diff
@@ -21,18 +21,18 @@ variable "instance_type" {
 }
 
 variable "max_size" {
-    default = 1
-    type = "number"
+    default = "3"
+    type = "string"
```

Comment:
Do these need to be a string? I know that we had set them to number originally so that we could type-check them a bit better.

Reply:
@gabrieljackson I set them to number the previous time thinking that terraform will accept it, but it gets the error that the value needs to be of type string. Its an AWS/Terraform thing and it accepts only strings...